### PR TITLE
Move last question in section, does not automatically create a new question

### PR DESCRIPTION
**Pull request by zimboflyman**
### What is the context of this PR?

If a section has only one question and that question is moved to a previous section - No new question is automatically created. 
this creates issues as sections are expected to have at least one question

This ticket is to fix that

### How to review 

Create 2 sections, A and B
move the only question from section B to section A
Section B should automatically create a new question so that the section is not left without a question

Codacy quality review does not like my destructing statement on line 65 of withMovePage.js
any ideas?

### CODE REVIEW

File: eq-author/src/App/page/Design/withMovePage.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import movePageMutation from "graphql/movePage.graphql";
 import fragment from "graphql/fragments/movePage.graphql";
 import { buildPagePath } from "utils/UrlUtils";
 import { remove } from "lodash";
+// import getNextPage from "utils/getNextOnDelete";
```
Comment (MattElcock) [inline eq-author/src/App/page/Design/withMovePage.js:6]: Rogue comment

File: eq-author/src/App/page/Design/withMovePage.test.js [hunk 2]
```diff
@@ -52,15 +72,16 @@ describe("withMovePage", () => {
   describe("mapMutateToProps", () => {
     beforeEach(() => {
       mutate = jest.fn(() => Promise.resolve(result));
+      // handleMove = jest.fn(() => Promise.resolve());
```
Comment (MattElcock) [inline eq-author/src/App/page/Design/withMovePage.test.js:75]: Rogue comment

File: eq-author/src/App/page/Design/withMovePage.js (hunk 1 above)
Comment (zimboflyman) [reply eq-author/src/App/page/Design/withMovePage.js]: aieeee - beat the developer!

File: eq-author/src/App/page/Design/withMovePage.test.js (hunk 2 above)
Comment (zimboflyman) [reply eq-author/src/App/page/Design/withMovePage.test.js]: beat him twice!!!

**Review by MattElcock — APPROVED** [review]

**Pull request merged**